modules/context/repo.go

```diff
@@ -309,11 +309,17 @@ func EarlyResponseForGoGetMeta(ctx *Context) {
 		ctx.PlainText(http.StatusBadRequest, "invalid repository path")
 		return
 	}
-	ctx.PlainText(http.StatusOK, com.Expand(`<meta name="go-import" content="{GoGetImport} git {CloneLink}">`,
+	res, err := vars.Expand(`<meta name="go-import" content="{GoGetImport} git {CloneLink}">`,
 		map[string]string{
 			"GoGetImport": ComposeGoGetImport(username, reponame),
 			"CloneLink":   repo_model.ComposeHTTPSCloneURL(username, reponame),
-		}))
+		})
+	if err != nil {
+		log.Error(err.Error())
+		ctx.PlainText(http.StatusInternalServerError, "expand vars failed")
+		return
+	}
+	ctx.PlainText(http.StatusOK, res)
```

Comment:
I just noticed: That whole operation is overkill. It can be greatly simplified:
```suggestion
	ctx.PlainText(http.StatusOK, fmt.Sprintf(`<meta name="go-import" content="%s git %s">`, ComposeGoGetImport(username, reponame), repo_model.ComposeHTTPSCloneURL(username, reponame)))
```